src/components/advanced-table/advanced-table.tsx

```diff
@@ -153,8 +155,35 @@ export function AdvancedTable<
     setMaxPage(newPage);
   };
 
+  const onLastPage = page === maxPage && nextToken === undefined;
+
   const height = props.height ?? 'auto';
 
+  /**
+   * Renders the label to the left of the pagination buttons.
+   */
+  const labelDisplayedRows = useCallback(
+    ({ from, to, count }: LabelDisplayedRowsArgs) => {
+      if (count === -1) {
+        const loadedRows = rows?.length ?? 0;
+        if (onLastPage) {
+          // for some reason `to` is set incorrectly on the last page in
+          // server-side pagination, so we need a custom template literal for
+          // this case.
+          return `${from}-${loadedRows} of ${loadedRows}`;
+        } else {
+          return (
+            `${from}-${to} of ${loadedRows}` +
```

Comment:
This should also use `trans.__()`. I don't think that the "25+", "50+" values are meaningful. The user already knows that there is a next page if the `>` button is active. I think that if the system doesn't provide a total count, we should just display `X–Y` instead of `X–Y of Y+`.

Reply:
I disagree. It's not clear with just `X-Y` that it's referring to the row count. For example, given a page size of 5 (e.g. in the detail views), it's not immediately obvious that the string `1-5` refers to the rows in my table. The user would have to count the 5 rows in the table and then infer "ah, this is referring to how many rows are being rendered by the table". `1-5 of 5+` makes this more clear.

Reply:
That's an interesting discussion. I believe both points of view are valid with concern that @dlqqq raised about @jweill-aws offer potentially solved by using `Rows X-Y` instead of just `X-Y`

Reply:
Let's discuss this as a team later, and we can open a follow-up PR if necessary.